tools/self-hosted-runner-utils/rerun_queued_workflows.py

```diff
@@ -0,0 +1,74 @@
+import os
+import argparse
+from datetime import datetime, timedelta
+
+from github import Github
+
+
+def pretty_time_delta(seconds):
+    # taken from https://gist.github.com/thatalextaylor/7408395
+    sign_string = '-' if seconds < 0 else ''
```

Comment:
Aren't we supposed to use double quotes, per https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#strings
```suggestion
    sign_string = "-" if seconds < 0 else ""
```